tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1206,10 +1206,11 @@ class CursorRangeInfoConsumer : public SwiftASTConsumer {
     // blocked waiting on the AST to be fully typechecked.
 
     ImmutableTextSnapshotRef InputSnap;
-    if (auto EditorDoc = Lang.getEditorDocuments()->findByPath(InputFile))
+    if (auto EditorDoc = Lang.getEditorDocuments()->findByPath(InputFile)) {
       InputSnap = EditorDoc->getLatestSnapshot();
       if (!InputSnap)
         return false;
```

Comment:
The original behaviour was correct, and the fix is we should re-indent this

```c++
if (auto EditorDoc = Lang.getEditorDocuments()->findByPath(InputFile))
  InputSnap = EditorDoc->getLatestSnapshot();
if (!InputSnap)
  return false;
```

This code used to look like that, but during a refactoring the indentation was incorrectly updated.

Reply:
Ah, okay.